src/tasks/TaskHelper.ts

```diff
@@ -107,6 +129,9 @@ export async function getAssociatedDockerBuildTask(runTask: DockerRunTask): Prom
 }
 
 export async function getOfficialBuildTaskForDockerfile(dockerfile: string, folder: WorkspaceFolder): Promise<Task | undefined> {
+    // TODO

```

Comment:
Need to fix this method. We no longer resolve the Dockerfile as part of `fetchTasks` so a different approach is needed to find the "official" `docker-build` task.